benchmarks/fwd_pointers/fwd_benchmarks.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import print_function
+
+import argparse
+import random
+from collections import defaultdict
+from datetime import datetime as dt
+from datetime import timedelta as td
+
+import numpy as np
+import pandas as pd
+from dateutil.rrule import rrule, MINUTELY
+
+import arctic
+from arctic import Arctic
+from arctic._config import FwPointersCfg
+# import matplotlib.pyplot as plt
+
+price_template = (800.0, 1200.0)
+
+ONE_MIN_ATTRIBUTES = {
+    'BID': price_template,
+    'BID_TWAP': price_template,
+    'ASK': price_template,
+    'ASK_TWAP': price_template,
+    'HIGH': price_template,
+    'LOW': price_template,
+    'CLOSE': price_template,
+    'TWAP': price_template,
+    'ASKSIZE': (0.0, 400.0),
+    'BIDSIZE': (0.0, 400.0),
+    'TICK_COUNT': (1.0, 50.0),
+    'VOLUME': (0.0, 1000.0),
+}
+
+APPEND_NROWS = 10
+
+
+class FwPointersCtx:
```

Comment:
Use new style classes: `class FwPointersCtx(object)`

Reply:
Yeah, I am used to the py3 style classes without the object, forgot about py2.